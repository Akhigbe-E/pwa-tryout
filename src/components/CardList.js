import React from "react";
import Card from "./Card";

export const CardList = ({ robots }) => {
  console.log(robots);
  return robots.map(({ id, name, email }) => (
    <Card
      id={id}
      name={name}
      email={email}
      key={id}
      src={`https://robohash.org/${id}?200x200`}
    />
  ));
};
