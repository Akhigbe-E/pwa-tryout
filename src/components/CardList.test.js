import React from "react";
import { shallow } from "enzyme";
import { CardList } from "./CardList";

it("test screenshot of CardList component", () => {
  const mockRobots = [
    {
      id: 1,
      name: "John Snow",
      username: "KingOfTheNorth",
      email: "john@north.com"
    }
  ];
  expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot();
});
