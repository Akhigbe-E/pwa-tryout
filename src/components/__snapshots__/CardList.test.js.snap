// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test screenshot of CardList component 1`] = `ShallowWrapper {}`;
